CSharp/DevVmPowershell/DevVmPsModules/Cmdlets/ConfirmInstallerAndInstanceRelativityVersionAreEqualModule.cs

```diff
@@ -66,13 +66,8 @@ protected override void ProcessRecordCode()
 			//Validate Input arguments
 			ValidateInputArguments();
 
-			IConnectionHelper connectionHelper = new ConnectionHelper(
-				relativityInstanceName: RelativityInstanceName,
-				relativityAdminUserName: RelativityAdminUserName,
-				relativityAdminPassword: RelativityAdminPassword,
-				sqlAdminUserName: SqlAdminUserName,
-				sqlAdminPassword: SqlAdminPassword);
-			IRelativityVersionHelper relativityVersionHelper = new RelativityVersionHelper(connectionHelper);
+			IRestHelper restHelper = new RestHelper();
+			IRelativityVersionHelper relativityVersionHelper = new RelativityVersionHelper(restHelper, RelativityInstanceName, RelativityAdminUserName, RelativityAdminPassword);
```

Comment:
Why are we not using the IConnectionHelper and passing the credentials directly to the RelativityVersionHelper constructor? Using IConnectionHelper helps with Dependency Injection.

- [ ] Use IConnectionHelper

Reply:
Done